roles/pulibrary.figgy/README.md

```diff
@@ -0,0 +1,48 @@
+Role Name
+=========
+
+A brief description of the role goes here.
+
+Requirements
+------------
+
+Any pre-requisites that may not be covered by Ansible itself or the role should
```

Comment:
Did we want to describe the additional items added in the playbooks for workers ect.?